src/lib/Form/Type/ContentType/ContentTypeUpdateType.php

```diff
@@ -115,6 +115,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('publishContentType', SubmitType::class, [
                 'label' => /** @Desc("OK") */ 'content_type.publish',
             ])
+            ->add('publishAndEditContentType', SubmitType::class, [
+                'label' => /** @Desc("Save") */ 'content_type.publish_and_edit',
```

Comment:
`save_no_close`? publishAndEdit seems a little bit strange to me, but do not change this on my opinion alone ;)

Or just `save`, as it seems it works that way in unified style across the system now.

Reply:
I believe it was because it is content, but for the user should not look like like the content. Not sure about what to choose here.